cookbook/configuration/web_server_configuration.rst

```diff
@@ -269,6 +269,8 @@ The **minimum configuration** to get your application running under Nginx is:
             fastcgi_split_path_info ^(.+\.php)(/.*)$;
             include fastcgi_params;
             fastcgi_param SCRIPT_FILENAME $document_root$fastcgi_script_name;
+            # symlink issue with PHP-FPM and OPcache. Symlink may not be refreshed, so it's necessary to use a real path (eg. after a deployment with Capifony which uses the current symlink to point towards a release directory, you may notice that PHP-FPM displays old pages or freezes) :
+            fastcgi_param DOCUMENT_ROOT $realpath_root;
```

Comment:
If I understand the issue correctly, I would propose to change this to something like this:

```
            # When you are using symlinks to link the document root to the
            # current version of your application, you should pass the real
            # application path instead of the path to the symlink to PHP
            # FPM.
            # Otherwise, PHP's OPcache may not properly detect changes to
            # your PHP files (see https://github.com/zendtech/ZendOptimizerPlus/issues/126
            # for more information).
            # fastcgi_param  SCRIPT_FILENAME  $realpath_root$fastcgi_script_name;
            # fastcgi_param  DOCUMENT_ROOT    $realpath_root;
```

But: Where is this `$realpath_root` variable configured? Is that something that is predefined by Nginx?